astropy/coordinates/tests/accuracy/test_altaz_icrs.py

```diff
@@ -91,6 +91,7 @@ def test_against_hor2eq():
     assert distance_noatm < 0.4 * u.arcsec
 
 
+@pytest.mark.skipif(reason="See https://github.com/astropy/astropy/issues/11305")
```

Comment:
I thought @saimn or @mhvk was against this? If not, I have no problem with this.